server/modules/record/repository/nodeRepository.js

```diff
@@ -172,29 +183,39 @@ export const updateNode = async (surveyId, nodeUuid, value, meta, draft, client
     date_modified = ${DbUtils.now}
     WHERE uuid = $3
     `,
-    [JSON.stringify(value), meta || {}, nodeUuid]
-  )
-  const node = await client.one(
-    `
-    ${_getNodeSelectQuery({ surveyId, draft })}
-    WHERE n.uuid = $1
-  `,
-    nodeUuid,
-    dbTransformCallback
+    [A.isEmpty(value) ? null : JSON.stringify(value), meta || {}, nodeUuid]
```

Comment:
I don't know if it's better to use 'default' or null in that case...

Reply:
lets see